### ISSUE TITLE
#126 maybe omit the 'self.point_attn_logits_scale'?

### ISSUE THREAD

**Issue by CiaoHe** [lucidrains/invariant-point-attention#1]
Hi luci:

I read the original paper and compare it to your implement, found one place might be some mistake:

#126. `attn_logits_points = -0.5 * (point_dist * point_weights).sum(dim = -1)`,

I thought it should be `attn_logits_points = -0.5 * (point_dist * point_weights * self.point_attn_logits_scale).sum(dim = -1)`

Thanks for your sharing!

**Comment by lucidrains** [comment]
@CiaoHe yes, you are right, fixed in 0.0.4! :pray:
